packages/synapse-react-client/src/components/widgets/RadioGroup.tsx

```diff
@@ -34,10 +34,10 @@ export function RadioGroup<T extends string | boolean | number = string>(
 export type RadioOptionProps<T extends string | boolean | number = string> = {
   label: React.ReactNode
   value: T
-  currentValue?: T
   style?: React.CSSProperties
   onChange: (value: T) => void
   disabled?: boolean
+  checked: boolean
```

Comment:
Replace `currentValue` with `checked`, which is a bit more intuitive/more closely matches the `input` interface